### PR TITLE
known issue with autopilot upgrades

### PR DESCRIPTION
**Pull request by JMGoldsmith**
This PR is to notify users of a known issue with autopilot upgrades.

### CODE REVIEW

File: website/content/partials/known-issues/autopilot-upgrade-upgrade-version.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### New nodes added by autopilot upgrades provisioned with the wrong version
+
+#### Affected versions
+
+- 1.15.3 - 1.15.8
+- 1.16.1 - 1.16.2
```
Comment (elliesterner) [inline website/content/partials/known-issues/autopilot-upgrade-upgrade-version.mdx:6]: what about 1.15.9 and 1.16.3?

Comment (JMGoldsmith) [reply website/content/partials/known-issues/autopilot-upgrade-upgrade-version.mdx:6]: Depending on release of those versions, won't this be patched? Or will it go in the next release after?

File: website/content/docs/upgrading/upgrade-to-1.15.x.mdx [hunk 2]
```diff
@@ -82,3 +82,5 @@ option.
 @include 'known-issues/perf-standbys-revert-to-standby.mdx'
 
 @include 'known-issues/1_13-reload-census-panic-standby.mdx'
+
+@include `known-issues/autopilot-upgrade-upgrade-version.mdx'
```
Comment (elliesterner) [inline website/content/docs/upgrading/upgrade-to-1.15.x.mdx:86]:
```suggestion
@include 'known-issues/autopilot-upgrade-upgrade-version.mdx'
```

File: website/content/docs/upgrading/upgrade-to-1.16.x.mdx [hunk 3]
```diff
@@ -101,3 +101,4 @@ more details on the configuration.
 
 @include 'known-issues/1_13-reload-census-panic-standby.mdx'
 
+@include `known-issues/autopilot-upgrade-upgrade-version.mdx'
```
Comment (elliesterner) [inline website/content/docs/upgrading/upgrade-to-1.16.x.mdx:104]:
```suggestion
@include 'known-issues/autopilot-upgrade-upgrade-version.mdx'
```

File: website/content/docs/upgrading/upgrade-to-1.15.x.mdx (hunk 2 above)
Comment (elliesterner) [reply website/content/docs/upgrading/upgrade-to-1.15.x.mdx:86]: if you check the vercel build, you can see that the partial wasn't getting loaded properly :) 
image

File: website/content/partials/known-issues/autopilot-upgrade-upgrade-version.mdx (hunk 1 above)
Comment (JMGoldsmith) [reply website/content/partials/known-issues/autopilot-upgrade-upgrade-version.mdx]: They did! updated.

File: website/content/docs/upgrading/upgrade-to-1.15.x.mdx (hunk 2 above)
Comment (JMGoldsmith) [reply website/content/docs/upgrading/upgrade-to-1.15.x.mdx:86]: Ahh, thanks, fixed.

**Review by elliesterner — APPROVED** [review]

**Pull request merged**